bin/submit.sh

```diff
@@ -31,12 +31,31 @@ else
   fi
 fi
 
+#0: inital value
+#1: deploy with assembly dist package file
+#2: deploy with project package
+CHUNJUN_DEPLOY_MODE=0
 if [[ $CHUNJUN_HOME && -z $CHUNJUN_HOME ]];then
-    export CHUNJUN_HOME=$CHUNJUN_HOME
+  export CHUNJUN_HOME=$CHUNJUN_HOME
 else
-    export CHUNJUN_HOME="$(cd "`dirname "$0"`"/../chunjun-dist; pwd)"
+  CHUNJUN_HOME="$(cd "`dirname "$0"`"/..; pwd)"
+  if [ -d "$CHUNJUN_HOME/chunjun-dist" ]; then
+    CHUNJUN_HOME="$CHUNJUN_HOME/chunjun-dist"
+    CHUNJUN_DEPLOY_MODE=2
```

Comment:
the sequence of CHUNJUN_DEPLOY_MODE should be consistent with the following code to make code more understandable. So the better sequence should be 1、2 ...